app/(auth)/sign-up-flow/phone.tsx

```
@@ -82,21 +82,16 @@ export default function PhoneScreen() {
           <CountryPicker
             countryCode={countryCode}
             onSelect={country => {
-              setCountryCode(country.cca2);
+              setCountryCode(country.cca2 as CountryCode);
               setCC(country.cca2);
               if (country.callingCode && country.callingCode[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)